docs/user-guide/templates/pipeline-templates.md

```diff
@@ -0,0 +1,194 @@
+---
+layout: main
+title: Pipeline Templates
+category: User Guide
+menu: menu
+toc:
+    - title: Pipeline Templates
+      url: "#pipeline-templates"
+    - title: Finding templates
+      url: "#finding-pipeline-templates"
+    - title: Using a template
+      url: "#using-a-pipeline-template"
+    - title: "Version/Tag Semantics"
+      url: "#versiontag-semantics"
+      subitem: true
+    - title: Creating a template
+      url: "#creating-a-template"
+    - title: Writing a template yaml
+      url: "#writing-a-template-yaml"
+      subitem: true
+    - title: Writing a screwdriver yaml
+      url: "#writing-a-screwdriver-yaml-for-your-template-repo"
+      subitem: true
+    - title: Validating templates
+      url: "#validating-templates"
+      subitem: level-2
+    - title: Tagging templates
+      url: "#tagging-templates"
+      subitem: level-2
+    - title: Testing a template
+      url: "#testing-a-template"
+    - title: Removing a template
+      url: "#removing-a-template"
+---
+# Pipeline Templates
+
+Pipeline Templates are snippets of predefined configuration that people can use to define a [screwdriver.yaml](../configuration). A pipeline template contains a series of predefined jobs with steps along with a selected container image.
+
+## Finding pipeline templates
+
+To figure out which templates already exist, you can make a `GET` call to the `/pipeline/templates` [API](../api) endpoint.
+
+
+## Using a pipeline template
+
+To use a template, define a `screwdriver.yaml` with a `template` key. In this example, we are using the [nodejs/test template](https://cd.screwdriver.cd/templates/nodejs/test).
```

Comment:
looks like the `nodejs/test template` is a job template, being put here mistakenly?